### PR TITLE
Added title to the home page and made it responsive

### PR DESCRIPTION
**Pull request by ojasaklechayt**
Issue Number: #49 

Description:
      I have done the following things:
        1. Added H1 tag which says " Welcome to Jarvis " which improves the UI
        2. Made it basic responsive
 
Screenshot: 
image

### CODE REVIEW

**Comment by J0SAL** [comment]
@ojasaklechayt, it's failing due to some reason, please resolve that.
I had already told you the trick to check the failing conditions.

**Comment by ojasaklechayt** [comment]
> @ojasaklechayt, it's failing due to some reason, please resolve that. I had already told you the trick to check the failing conditions.

I did npm run build but it was going correct

**Comment by ojasaklechayt** [comment]
@J0SAL

File: frontend/components/transactions/DownloadTransactions.tsx [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import React from "react";
-import { Dropdown } from "react-bootstrap";
+import { Button } from "react-bootstrap";
```
Comment (J0SAL) [inline frontend/components/transactions/DownloadTransactions.tsx:2]: ??

File: frontend/next.config.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
-}
-
-module.exports = nextConfig
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
+  swcMinify: true,
+}
+
+module.exports = nextConfig
```
Comment (J0SAL) [inline frontend/next.config.js:7]: undo these

File: frontend/pages/_app.tsx [hunk 3]
```diff
@@ -1,37 +1,37 @@
...
+                <Component {...pageProps} />
+                <Footer />
+              </React.Fragment>
+            </DataState>
+          </SpeechProvider>
+        </NotificationProvider>
+      </MoralisProvider>
+    </React.StrictMode>
+  );
+}
```
Comment (J0SAL) [inline frontend/pages/_app.tsx:37]: what's the change here?

File: frontend/pages/api/hello.ts [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
+type Data = {
+  name: string
+}
+
+export default function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Data>
+) {
+  res.status(200).json({ name: 'John Doe' })
+}
```
Comment (J0SAL) [inline frontend/pages/api/hello.ts:13]: undo these

File: frontend/pages/index.tsx [hunk 5]
```diff
@@ -1,72 +1,72 @@
...
+          borderRadius: "30px",
+        }}
+        onClick={scrollToTop}
+        hidden={!showButton}
+      >
+        ^
+      </button>
+    </div>
+  );
+}
```
Comment (J0SAL) [inline frontend/pages/index.tsx:72]: ??

File: frontend/styles/Home.module.css [hunk 6]
```diff
@@ -1,129 +1,129 @@
...
+  .footer {
+    border-color: #222;
+  }
+  .code {
+    background: #111;
+  }
+  .logo img {
+    filter: invert(1);
+  }
+}
```
Comment (J0SAL) [inline frontend/styles/Home.module.css:129]: ??

File: frontend/styles/globals.css [hunk 7]
```diff
@@ -1,40 +1,40 @@
...
+  body {
+    color: white;
+    background: black;
+  }
+}
+
+/* media query for mobile */
+@media only screen and (max-width: 600px) {
+
 }
```
Comment (J0SAL) [inline frontend/styles/globals.css:40]: ??

**Review by J0SAL — CHANGES_REQUESTED** [review]
There are a lot of problems with the PR, I have either requested changes or marked with the *??*

few problems are, 
- code is not sync
- the whole file shouldn't be changed in case if just a line is changed
- don't make indentation changes to the files

To resolve these issues, my advice would be to create a new PR(with a new fork/updated code) and make specific changes...
you can ask @Abhijit25Mishra for help

**Comment by ojasaklechayt** [comment]
All these are due to the conflicts as soon as I merge these. Some other sync appears which cause the merge conflicts

**Comment by J0SAL** [comment]
> There are a lot of problems with the PR, I have either requested changes or marked with the _??_
> 
> few problems are,
> 
> * code is not sync
> * the whole file shouldn't be changed in case if just a line is changed
> * don't make indentation changes to the files
> 
> To resolve these issues, my advice would be to create a new PR(with a new fork/updated code) and make specific changes... you can ask @Abhijit25Mishra for help

Like I said here, your changes are undoing the previously updated changes, and there are really many unwanted changes... best thing would be to create a new PR, which will work on the code from the most recent main branch

**Comment by ojasaklechayt** [comment]
> > There are a lot of problems with the PR, I have either requested changes or marked with the _??_
> > few problems are,
> > 
> > * code is not sync
> > * the whole file shouldn't be changed in case if just a line is changed
> > * don't make indentation changes to the files
> > 
> > To resolve these issues, my advice would be to create a new PR(with a new fork/updated code) and make specific changes... you can ask @Abhijit25Mishra for help
> 
> Like I said here, your changes are undoing the previously updated changes, and there are really many unwanted changes... best thing would be to create a new PR, which will work on the code from the most recent main branch

okay

**Pull request closed without merging**